aten/src/THCUNN/generic/SpatialFullDilatedConvolution.cu

```diff
@@ -332,23 +339,36 @@ void THNN_(SpatialFullDilatedConvolution_accGradParameters)(
            accreal scale_)
 {
   real scale = ScalarConvert<accreal, real>::to(scale_);
-  int nInputPlane = THCTensor_(size)(state, gradWeight, 0);
-  int nOutputPlane = THCTensor_(size)(state, gradWeight, 1);
-
   THCUNN_assertSameGPU(state, 6, input, gradOutput, gradWeight,
                        gradBias, columns, ones);
   THNN_(SpatialFullDilatedConvolution_shapeCheck)
-       (state, input, gradOutput, gradWeight, gradBias, kH, kW, dH, dW, padH, padW, dilationH, dilationW, adjH, adjW);
+       (state, input, gradOutput, gradWeight, gradBias, kH, kW, dH, dW,
+        padH, padW, dilationH, dilationW, adjH, adjW, 1);
+
+  int nOutputPlane;
+  if (gradWeight != NULL) {
+    nOutputPlane = THCTensor_(size)(state, gradWeight, 1);
+  } else if (gradBias != NULL) {
+    nOutputPlane = THCTensor_(size)(state, gradBias, 0);
+  } else {
+    return;
```

Comment:
Uh.... return?!

Reply:
Yeah, if both are NULL then there is no parameter to accumulate gradient!